Registry/application_lifecycle_events.setreg

```diff
@@ -7,24 +7,24 @@
 // related to the event
 {
     "O3DE" : {
-        "Runtime": {
-            "Application": {
-                "LifecycleEvents": {
-                    "SystemComponentsActivated": {},
-                    "SystemComponentsDeactivated": {},
-                    "ReflectionManagerAvailable": {},
-                    "ReflectionManagerUnavailable": {},
-                    "SystemAllocatorCreated": {},
-                    "SystemAllocatorPendingDestruction": {},
-                    "SettingsRegistryAvailable": {},
-                    "SettingsRegistryUnavailable": {},
-                    "ConsoleAvailable": {},
-                    "ConsoleUnavailable": {},
-                    "GemsLoaded": {},
-                    "GemsUnloaded": {},
-                    "FileIOAvailable": {},
-                    "FileIOUnavailable": {}
-                }
+        "Application": {
+            "LifecycleEvents": {
+                "SystemComponentsActivated": {},
+                "SystemComponentsDeactivated": {},
+                "ReflectionManagerAvailable": {},
+                "ReflectionManagerUnavailable": {},
+                "SystemAllocatorCreated": {},
+                "SystemAllocatorPendingDestruction": {},
+                "SettingsRegistryAvailable": {},
+                "SettingsRegistryUnavailable": {},
+                "ConsoleAvailable": {},
+                "ConsoleUnavailable": {},
+                "GemsLoaded": {},
+                "GemsUnloaded": {},
+                "FileIOAvailable": {},
+                "FileIOUnavailable": {},
+                "SystemEntityActivated": {},
+                "SystemInterfaceCreated": {}
```

Comment:
Can we call this `CrySystemInterfaceCreated` or "LegacySystemInterfaceCreated"?
I know @bosnichd is still trying to redcode "CrySystem".
That would distinguish it from the non-Cry lifecycle event.s

Reply:
We don't actually users to start adding more uses of this event.

Reply:
Changed it to LegacySystemInterfaceCreated